compiler/rustc_middle/src/ty/context.rs

```diff
@@ -558,6 +558,8 @@ pub struct GlobalCtxt<'tcx> {
 
     /// Stores memory for globals (statics/consts).
     pub(crate) alloc_map: Lock<interpret::AllocMap<'tcx>>,
+
+    pub inlined_internal_defs: Lock<FxHashSet<LocalDefId>>,
```

Comment:
This field will need to be removed before merging, for incr. comp. support.

Reply:
Why?

The most recent commit in here is a hack to make crater exercise this PR without having to do a `--release` run.

Reply:
In incremental mode, optimized MIR may get loaded from the on-disk cache, and the insertions into this map may not be re-executed.